pkg/server/apiHandlerv2.go

```diff
@@ -672,3 +676,303 @@ func (h *apiHandlerv2) uuid(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write(cloudEnvelope)
 }
+
+func extractQuoteAttestTemplate(quote *attest.ZAttestQuote) *common.PCRTemplate {
```

Comment:
No objection to your moving all of this out of `apiHandler.go`, as it only applies to v2, but I think this complex attestation stuff might be better served in its own `attestation.go` file.

Reply:
Sounds reasonable, moved, it is large part of code for attestation only functions.